observability/otel/otel-collector/otel-collector-receivers.adoc

```diff
@@ -864,6 +864,95 @@ rules:
 <1> The service account of the Collector that has the required ClusterRole `otel-collector` RBAC.
 <2> The list of namespaces to collect events from. The default value is empty, which means that all namespaces are collected.
 
+[id="aws-xray-exporter_{context}"]
+== AWS X-Ray Exporter
+
+The AWS X-Ray exporter converts OpenTelemetry spans to AWS X-Ray Segment Documents and then sends them directly to X-Ray using the PutTraceSegments API.
+
+:FeatureName: The AWS X-Ray exporter
+include::snippets/technology-preview.adoc[]
+
+.OpenTelemetry Collector custom resource with the enabled AWS X-Ray Exporter
+[source,yaml]
+----
+# ...
+  config:
+    exporters:
+      awsxray:
+        region: eu-west-1 # <1>
+        resource_arn: "arn:aws:ec2:us-east1:123456789:instance/i-293hiuhe0u" # <2>
+        role_arn: "arn:aws:iam::123456789:role/monitoring-EKS-NodeInstanceRole" # <3>
+        indexed_attributes: [ "indexed_attr_0", "indexed_attr_1" ] # <4>
+        aws_log_groups: ["group1", "group2"] # <5>
+        request_timeout_seconds: 120 # <6>
+# ...
+----
+<1> Send segments to AWS X-Ray service in a specific region.
+<2> Amazon Resource Name (ARN) of the AWS resource running the collector.
+<3> IAM role to upload segments to a different account.
+<4> List of attribute names to be converted to X-Ray annotations.
+<5> List of log group names for CloudWatch.
+<6> Number of seconds before timing out a request. If omitted, the default value is `+30+`.
+
+[id="aws-cloudwatch-exporter_{context}"]
+== AWS CloudWatch Exporter
+
+The AWS CloudWatch exporter sends logs data to link:https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/WhatIsCloudWatchLogs.html[AWS CloudWatch Logs].
+AWS credentials are retrieved from the link:https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials[default credential chain].
```

Comment:
@pavolloffay, questio, there are docs for `v1` and `v2`. They recommend `v2`. Do you still prefer to document `v1`? (I don't care, just want to clarify this).

Reply:
The exporter is using v1 https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/awscloudwatchlogsexporter/go.mod#L6

Reply:
I'll keep the v2 link but just comment it out until further notice.